zhaquirks/thirdreality/night_light.py

```diff
@@ -0,0 +1,104 @@
+"""Third Reality night light zigbee devices."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    MultistateInput,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks import LocalDataCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.thirdreality import THIRD_REALITY
+
+ThirdReality_Specific_CLUSTER_ID = 0xFC00
+
+
+class ThirdRealitySpecificCluster(CustomCluster):
+    """privatecluster, only used to relay ias zone information to ias zone Cluster."""
+
+    cluster_id = ThirdReality_Specific_CLUSTER_ID
+
+    def _update_attribute(self, attrid, value):
+        super()._update_attribute(attrid, value)
+        self.endpoint.ias_zone.update_attribute(
+            IasZone.AttributeDefs.zone_status.id, value
+        )
```

Comment:
Please check that `attrid` matches `0x0002` before forwarding to the `ias_zone` cluster.
Otherwise, this *will* break when another attribute is used in the future. Not all HA users might update their HA version, so if they have updated devices/firmware, but outdated HA, this will break.

So please add a check that's something like this:
```suggestion
        if attrid == 0x0002:
            self.endpoint.ias_zone.update_attribute(
                IasZone.AttributeDefs.zone_status.id, value
            )
```

Relevant comments:
- https://github.com/zigpy/zha-device-handlers/pull/2509#discussion_r1287281874
- https://github.com/zigpy/zha-device-handlers/pull/2509#issuecomment-1676580102